packages/opencensus-web-core/src/trace/model/tracer.ts

```diff
@@ -22,62 +22,85 @@ import { randomTraceId } from '../../common/id-util';
 
 /** Tracer manages the current root span and trace header propagation. */
 export class Tracer extends TracerBase implements webTypes.Tracer {
+  private static singletonInstance: Tracer;
+
+  /** Gets the tracer instance. */
+  static get instance(): Tracer {
```

Comment:
What's the motivation for adding this `instance` getter?

Reply:
I saw that opencensus node does it and I think is more understandable that `Tracer` is a Singleton class.

Reply:
OK, makes sense.